xija/model.py

```diff
@@ -88,19 +91,22 @@ def __init__(self, name=None, start=None, stop=None, dt=328.0,
             stop = stop or model_spec['datestop']
             start = start or model_spec['datestart']
             name = name or model_spec['name']
+            dt = dt or model_spec['dt']
 
         if stop is None:
             stop = DateTime() - 30
         if start is None:
             start = DateTime(stop) - 45
         if name is None:
             name = 'xijamodel'
+        if dt is None:
+            dt = DEFAULT_DT
 
         self.name = name
         self.comp = OrderedDict()
-        self.dt = dt
+        self.dt = self._check_timestep(dt)
```

Comment:
`check_timestep` implies that it is running a check that might succeed or fail, instead this is actually getting an allowed timestep.  So I'd suggest `_get_allowed_timestep()`.  With that I'm good with this.